e2e/core/viewer/viewer-component.e2e.ts

```diff
@@ -452,4 +455,45 @@ xdescribe('Viewer', () => {
             });
         });
     });
+
+    describe('Code viewer', () => {
```

Comment:
the name of this describe should be something that highlight the thing that we are testing an extension